libs/utils/energy_model.py

```diff
@@ -774,7 +774,105 @@ def _find_core_groups(cls, target):
         return sorted(ret, key=lambda x: x[0])
 
     @classmethod
-    def from_target(cls, target):
+    def from_simplifiedEM_target(cls, target):
+        """
+        Create an EnergyModel by reading a target filesystem on a device with
+        the new Simplified Energy Model present.
+
+        This uses the energy_model sysctl added by EAS patches to exposes
+        the frequency domains, together with a tuple of capacity, frequency
+        and active power for each CPU. This feature is not upstream in mainline
+        Linux (as of v4.17), and only exists in Android kernels later than
+        android-4.14.
+
+        Wrt. idle states - the EnergyModel constructed won't be aware of
+        any power data or topological dependencies for entering "cluster"
+        idle states since the simplified model has no such concept.
+
+        Initialises only Active States for CPUs and clears all other levels.
+
+        :param target: Devlib target object to read filesystem from. Must have
+                       cpufreq and cpuidle modules enabled.
+        :returns: Constructed EnergyModel object based on the parameters
+                  reported by the target.
+        """
+        if 'cpufreq' not in target.modules:
```

Comment:
Why do we need the `cpufreq` module ? The frequency exposed directly in the EM should be enough no ?

Reply:
Yes, actually we don't need cpufreq for the simplified EM loading, only for the traditional model.

Reply:
Done